csatar-plugins/forms/traits/AjaxControllerSimple.php

```diff
@@ -355,7 +524,9 @@ public function onSave()
         $attributeNames = [];
 
         foreach ($config->fields as $key => $value) {
-            $attributeNames[$key] = Lang::get($value['label']);
+            if ($value['type'] !== 'section') {
```

Comment:
Will there be always a $value['type']? 
As I remember this function is used only when added a new relation in edit mode, why do we need this change?